src/k8s/pkg/component/component_test.go

```diff
@@ -180,7 +172,7 @@ func TestNewHelmClientWithValidConfig(t *testing.T) {
 	g.Expect(mockHelmClient).ToNot(BeNil())
 	g.Expect(mockHelmClient).To(BeAssignableToTypeOf(&helmClient{}))
 
-	g.Expect(mockHelmClient.initializer).To(BeAssignableToTypeOf(&MockHelmClientInitializer{}))
+	g.Expect(mockHelmClient.initializer).To(BeAssignableToTypeOf(&mocks.HelmClientInitializer{}))
 	g.Expect(mockHelmClient.snap).To(BeAssignableToTypeOf(&mock.Snap{}))
 	g.Expect(mockHelmClient.config).To(HaveLen(3))
```

Comment:
Are these useful tests? What do we validate here?

Reply:
I'll remove this one.